packages/dds/tree/src/core/rebase/changeRebaser.ts

```diff
@@ -111,11 +120,13 @@ export function tagChange<T>(
 export function tagInverse<T>(
```

Comment:
I think we could rename this to something like `tagRollbackInverse` and get rid of the `isRollback` parameter.